docker/docker-compose.yml

```
@@ -1,7 +1,7 @@
 version: '3.2'
 services:
   graph-node:
-    image: graphprotocol/graph-node:v0.24.2
+    image: graphprotocol/graph-node:v0.25.0
```

Unfortunately I think this breaks our current subgraphs (and they are not sufficiently tested).
```suggestion
    image: graphprotocol/graph-node:v0.24.2
```

---

oh ok no problem